spiceypy/utils/support_types.py

```diff
@@ -66,122 +66,48 @@
 
 import numpy
 from numpy import ctypeslib as numpc
-import spiceypy.utils.exception_lookups as exception_lookups
-
-# Collection of supporting functions for wrapper functions
-__author__ = "AndrewAnnex"
-
-errorformat = """
-================================================================================
-
-Toolkit version: {tkvsn}
 
-{short} --
-{explain}
-{long}
+from .exceptions import (
+    SpiceyError,
+    SpiceyPyError,
+    NotFoundError,
+    SpiceyPyIOError,
+    SpiceyPyMemoryError,
+    SpiceyPyTypeError,
+    SpiceyPyKeyError,
+    SpiceyPyIndexError,
+    SpiceyPyRuntimeError,
+    SpiceyPyZeroDivisionError,
+    SpiceyPyValueError,
+    exceptions,
+)
 
-{traceback}
 
-================================================================================\
-"""
+# Collection of supporting functions for wrapper functions
+__author__ = "AndrewAnnex"
```

Comment:
I was really curious about the `__author__` metadata. I knew Python had this capability but had never really explored it. There's kind of an interesting debate on Stack Overflow about whether one should use these (other than in `setup.py`):

https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files#

Reply:
yeah I think it was actually auto-populated by pycharm years ago when I started the project and I've never bothered to remove it and kept the convention. I think some of these metadata, in particular the version are worth having.